### PR TITLE
add tendrl qe_evidence playbook

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

**Comment by mkudlej** [comment]
LGTM

**Comment by mkudlej** [comment]
Related pull request https://github.com/Tendrl/usmqe-setup/pull/32/files

File: qe_evidence.tendrl.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# one needs to tweak the list further - copy this into new playbook and just
+# comment out or add directories or files as you like.
+
+- name: Download evidence available on all machines
+  hosts: all
+  user: root
+  vars:
+    evidence_dirs:
+     - /var/log/audit/
+     - /var/log/salt/
```
Comment (mbukatov) [inline qe_evidence.tendrl.yml]: Do we really need to fetch salt logs here?

Comment (fbalak) [inline qe_evidence.tendrl.yml]: Probably not. I will remove it.

File: qe_evidence.tendrl.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  roles:
+    - qe-evidence
+    - qe-evidence-probe
+
+- name: Download evidence from Tendrl server machine
+  hosts: usm_server
+  user: root
+  vars:
+    evidence_dirs:
+     - /var/log/httpd/
```
Comment (ltrilety) [inline qe_evidence.tendrl.yml]: please add here /etc/tendrl/ directory

**Pull request merged**